RELEASENOTES.md

```diff
@@ -40,6 +40,8 @@
     *   Propagate ID3 `TCON` frame to `MediaMetadata.genre`
         ([#1305](https://github.com/androidx/media/issues/1305)).
 *   Image:
+    *   Add support for non-square DASH thumbnail grids
+        ([#1255](https://github.com/androidx/media/pull/1300)).
```

Comment:
@microkatz The PR number here doesn't match the actual PR (and issue 1255 is unrelated to this PR)